### ISSUE TITLE
WASM memory.copy instruction unimplemented 

### ISSUE THREAD

**Issue by schmatz** [Rerumu/Wasynth#22]
Trying to compile CPython for a challenge and see the error 
```
thread 'main' panicked at 'Unsupported instruction: MemoryCopy { src: 0, dst: 0 }', wasm-ast/src/factory.rs:616:18
```
[The spec](https://webassembly.github.io/spec/core/exec/instructions.html#exec-memory-copy) has more details about these instructions.

For reference, the wasm file is here: [python.wasm.zip](https://github.com/Rerumu/Wasynth/files/9428975/python.wasm.zip)

Might have a go at implementing this if I have some spare time.

**Comment by schmatz** [comment]
Well, I ended up implemented a PR and using that wasynth hilariously generates 47MiB of Luau code and it doesn't immediately crash when I run it. It doesn't output anything (my base wasm file doesn't run properly in any case) so I have no confirmation that it works, but at least it seems to do something.

**Comment by Rerumu** [comment]
This was finally fixed in 4bd5fb8d439c4763f3d7b7767400529c894d0b3c.
